pkg/runtime/cache/namespace.go

```diff
@@ -177,6 +201,17 @@ func (c *NamespaceCache) setNamespaceInfoFromK8sObject(ns *corev1.Namespace) {
 	if ok {
 		nsInfo.endpointURL = EndpointURL
 	}
+
+	nsInfo.deletionPolicies = map[string]string{}
+	nsDeletionPolicySuffix := "." + ackv1alpha1.AnnotationDeletionPolicy
+	for key, elem := range nsa {
+		if !strings.HasSuffix(key, nsDeletionPolicySuffix) {
+			continue
+		}
+		service := strings.TrimSuffix(key, nsDeletionPolicySuffix)
+		nsInfo.deletionPolicies[service] = elem
```

Comment:
I think we can apply a `string.ToLower` to `service` variable here. Thinking users can provide an `eCR` or `Eks` ...

Reply:
I also see that you're using `strings.ToLower(service)` to query the deletion policy